trunk/SUAVE/Input_Output/OpenVSP/vsp_read.py

```diff
@@ -169,4 +176,47 @@ def vsp_read(tag, units_type='SI'):
 		wing = vsp_read_wing(wing_id, units_type)
 		vehicle.append_component(wing)		
 	
+	# Initialize rotor network elements
+	number_of_lift_rotor_engines = 0
+	number_of_propeller_engines  = 0
+	lift_rotors = Data()
+	propellers  = Data()
+	
+	for prop_id in vsp_props:
+		prop = vsp_read_propeller(prop_id,units_type)
+		prop.tag = vsp.GetGeomName(prop_id)
+		if prop.orientation_euler_angles[1] >= 70 * Units.degrees:
+			lift_rotors.append(prop)
+			number_of_lift_rotor_engines += 1 
+		else:
+			propellers.append(prop)
+			number_of_propeller_engines += 1 
+
+	# Create the rotor network
+	if number_of_lift_rotor_engines>0 and number_of_propeller_engines>0:
+		# Lift + Cruise network
+		net = Lift_Cruise()
+		for i in range(number_of_lift_rotor_engines):
+			net.lift_rotors.append(propellers[list(lift_rotors.keys())[i]])
+		net.number_of_lift_rotor_engines = number_of_lift_rotor_engines	
+		
+		for i in range(number_of_propeller_engines):
+			net.propellers.append(propellers[list(propellers.keys())[i]])
+		net.number_of_propeller_engines = number_of_propeller_engines		
+		
+	else:
+		# Append all rotors as propellers for the battery propeller network
+		net = Battery_Propeller()
```

Comment:
is there a way to pass the type of network the user wants to specify into the vsp_read function? That way we do not assume it is a battery propeller.

Reply:
Updated to add a check for specified network; uses this convention if no network is specified